site/releases/2024-oct-22/release-notes.qmd

```diff
@@ -0,0 +1,513 @@
+
+result = run_test(
+    test_id="validmind.data_validation.DatasetDescription",
+    inputs={
+        "dataset": {
+            "input_id": "my_dataset",
+            "columns": ["col1"]
+        }
+    },
+)
+```
+
+Notice how the `dataset` input is set to a dictionary whose key `input_id` maps to a particular input dataset. And whose `columns` key sets the subset of columns from the original dataset to pass to the `DatasetDescription` test.
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+A new Jupyter Notebook demonstrates how to selectively include columns in your analysis based on your specific criteria.
+:::
+
+::: {.w-30-ns .tr}
+[Try it on JupyterHub](https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/how_to/filter_input_columns.ipynb){.button}
+:::
+
+::::
+
+
+
+
+
+## Enhancements
+
+<!---
+PR #996: Added page and api call to handle unverified emails
+URL: https://github.com/validmind/frontend/pull/996
+Labels: enhancement
+--->
+### Email verification improvements
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+This update enhances the user experience by introducing a new email verification error page. If you need to verify your email address but have been unable to do so, this page allows you to resend the verification email and navigate to the dashboard to access ValidMind once verification is complete. 
+:::
+
+::: {.w-30-ns .tr}
+[Learn about registering {{< fa book-open >}}](/guide/configuration/register-with-validmind.qmd){.button}
+:::
+
+::::
+
+<!---
+PR #905: Custom Field Type Code
+URL: https://github.com/validmind/frontend/pull/905
+Labels: enhancement
+--->
+### Enhanced custom field management with Python support
+
+This update enhances the custom field functionality, making it possible to define and test custom Python code to calculate an output. 
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+The results of your code execution are displayed clearly, and you can test your custom code using a new testing feature directly within the interface.
+:::
+
+::: {.w-40-ns .tr}
+[Learn about model inventory fields {{< fa book-open >}}](/guide/model-inventory/manage-model-inventory-fields.qmd){.button}
+:::
+
+::::
+
+The custom field editor has also been upgraded, now including a code editor and testing area, along with a drag-and-drop interface for selecting fields. The overall user experience has been improved with better error handling and feedback. Additionally, managing text fields is now more intuitive with expandable text displays, making it simpler to navigate and modify custom fields.
+
+Watch the demo:
+
+<div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/4d7d14b21d17416caefddf2b46fe05af?sid=70f2acd6-9a29-4856-a6ed-b3a0b28b084b" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
```

Comment:
@gtagle you had created this demo video for custom field management that seems helpful — are you OK with us including this in the release notes?